DesignSafe-CI/portal
# Task/DES-1990 - Bug fixes for None projects and project management modal

## Overview: ##
This PR fixes some bugs related to projects with "None" set as a project type (newly created projects). 

## PR Status: ##


## Related Jira tickets: ##

* [DES-1990](https://jira.tacc.utexas.edu/browse/DES-1990)

## Summary of Changes: ##
1. Fixed up loading/submitting conditions for project management modal (all projects)
2. Fixed modal freeze when saving new users (none projects)
3. Fixed redirect when selecting curation/publication preview (none projects)


## Testing Steps: ##
1. Create and update a project using the project management modal (all projects). This should display a "saving" or "creating" message and disable the form
2. Attempt to add a user to a project with no project type set
3. From within a project with no project type set, select the curation and publication preview buttons. These should open the "Manage Project" modal.
